spbuTest1/main.c

```diff
@@ -0,0 +1,48 @@
+#include <stdio.h>
+#include <stdbool.h>
+
+long long int FibonacciNumber(long long int theFirstNumber, long long int theSecondNumber, long long int sum)
+{
+    long long int nextNumber = theFirstNumber + theSecondNumber;
+    while (nextNumber <= 1000000)
+    {
+        if (nextNumber % 2 == 0)
+        {
+            sum += nextNumber;
+        }
+
+        theFirstNumber = theSecondNumber;
+        theSecondNumber = nextNumber;
+        nextNumber = theFirstNumber + theSecondNumber;
+    }
+
+    return sum;
+}
+
+long long int test()

```

Comment:
```suggestion
long long int test(void)
```
Иначе это функция с неопределённым количеством аргументов, немного не то, что тут хочется